src/web-ui/src/public/Checkout.vue

```diff
@@ -262,6 +187,15 @@ export default {
           this.$router.push('/');
         });
      })
+    },
+    async triggerAbandonedCartEmail () {
+      if (this.cart && this.cart.items.length > 0 ){
+        const cartItem = await this.getProductByID(this.cart.items[0].product_id)
+        AnalyticsHandler.recordAbanonedCartEvent(this.user,this.cart,cartItem)
+      }
+      else{
+        console.error("No items to export")
+      }
     }
   },
   computed: {
```

Comment:
Let's use the existing getters in `vuex` to fetch cart quantities. You'll notice that some calculations like tax are not available - these have been removed on the cart page and can also be removed here.